### ISSUE TITLE
Duckling Parse Date Time Range 

### ISSUE THREAD

**Issue by souvikg10** [facebook/duckling#186]
I am using the NL rules and I have also checked for English, whenever I parse a day/date, I receive the following output
```sh
[
    {
        "body": "30th april",
        "start": 0,
        "value": {
            "values": [
                {
                    "value": "2019-04-30T00:00:00.000-07:00",
                    "grain": "day",
                    "type": "value"
                },
                {
                    "value": "2020-04-30T00:00:00.000-07:00",
                    "grain": "day",
                    "type": "value"
                },
                {
                    "value": "2021-04-30T00:00:00.000-07:00",
                    "grain": "day",
                    "type": "value"
                }
            ],
            "value": "2019-04-30T00:00:00.000-07:00",
            "grain": "day",
            "type": "value"
        },
        "end": 10,
        "dim": "time",
        "latent": false
    }
```
In the values, I only get the future dates but it would also be useful to have the past ranges as well.

Do you think it is a nice idea. because for us when we ask our users, can you tell us the date of your transfer and user replies with 30th April, a range of past dates would be useful as well. 

Same applies when user says Monday.

**Comment by RomainWilbert** [comment]
Some libraries can be configured with a temporal context (future, past) which helps to decide in case of ambiguous results. This would be nice add to the options ?

**Comment by souvikg10** [comment]
Indeed. I would rather like the range from past to future if a date exact is not mentioned. For example if the user says Vorige maandag then it is previous monday but if the user says just maandag, i don't understand why it is assumed to be in the future. Time dimension seems to big for me to change this by myself. Maybe if someone can let me know where exactly are such things are handled in the code, i can go ahead and change them.

**Comment by Downchuck** [comment]
Certainly seems more reasonable for it to have prior april, current april and future april, than three future years of april.

**Comment by tbaptista** [comment]
I believe this would be solved if issue #29 gets completely resolved.

**Comment by patapizza** [comment]
Indeed, the default Duckling assumption is to return values in the future, unless the past is explicitly mentioned. As @tbaptista mentioned, this would be resolved with #29.

**Comment by cwying817** [comment]
Can this problem be resolved with #29 if duckling is running in docker(duckling http server)?
